inc/cleantalk-common.php

```diff
@@ -183,6 +176,16 @@ function apbct_base_call($params = array(), $reg_flag = false)
         'submit_time' => apbct_get_submit_time()
     );
 
+    /**
+     * Add exception_action if exists in params
+     */
+    if (
```

Comment:
Second part of condition
`isset($params['exception_action']) && $default_params['exception_action'] == 1`
will be always false. What is the point of it?

Reply:
See now in commit https://github.com/CleanTalk/wordpress-antispam/pull/74/commits/a173f179a665b57766a078397373817775672fa1

Reply:
This was a typo on the variable name typing.